docs/runtime-upgrades-and-versioning.md

```diff
@@ -0,0 +1,54 @@
+# Overview 
+
+This diagram shows our compile time and runtime dependencies to allow to guide upgrade process
+
+```plantuml
+@startuml
+
+frame "GitHub" as github {
+    folder "Substrate v0.9.*" as substrate
+    folder "Cumulus v0.9.*" as cumulus
+    folder "Polkadot v0.9.*" as polkadot
+    folder "ORML v0.x.y" as orml 
+    folder "Composable v.a.b" as composable 
```

Comment:
What is our versioning scheme normally? How do we decide on versions?

Reply:
I do not know. We have version bump for runtime, not sure if we follow and compile time pattern. So it is not important here as here is dependencies are important.

Reply:
added comment

Reply:
closing for period of inactivity. feel free to reopen